app/controllers/idv/link_sent_controller.rb

```diff
@@ -91,18 +90,5 @@ def document_capture_session_result
           document_capture_session&.load_doc_auth_async_result
       end
     end
-
-    def extend_timeout_using_meta_refresh
-      max_10min_refreshes = IdentityConfig.store.doc_auth_extend_timeout_by_minutes / 10
```

Comment:
This config `doc_auth_extend_timeout_by_minutes` looks to be used now, should it also be removed?